composer.json

```diff
@@ -16,7 +16,8 @@
         "symfony/framework-bundle": "^2.8|^3.0",
         "sonata-project/doctrine-phpcr-admin-bundle": "^2.0",
         "sonata-project/admin-bundle": "^3.6.0",
-        "symfony-cmf/tree-browser-bundle": "^2.0"
+        "symfony-cmf/tree-browser-bundle": "^2.0",
+        "egeloen/ckeditor-bundle": "^4.0"
```

Comment:
mhh .. can be on dev and into suggested then. The version comes by a simple `composer require` call.